SYCL/Basic/alloc_pinned_host_memory.cpp

```diff
@@ -0,0 +1,47 @@
+// REQUIRES: level_zero
+
+// RUN: %clangxx %s -o %t1.out -lsycl -I %sycl_include
+// RUN: %HOST_RUN_PLACEHOLDER %t1.out
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t2.out
+// RUN: env SYCL_PI_TRACE=2 ZE_DEBUG=1 %GPU_RUN_PLACEHOLDER %t2.out 2>&1 %GPU_CHECK_PLACEHOLDER
+
+// RUN: %HOST_RUN_PLACEHOLDER %t2.out
+// RUN: %CPU_RUN_PLACEHOLDER %t2.out
+// RUN: %GPU_RUN_PLACEHOLDER %t2.out
+// RUN: %ACC_RUN_PLACEHOLDER %t2.out
+// UNSUPPORTED: cl_options
```

Comment:
That causes the test skipped on windows.
It is better to split it in 2:
 - the one with host only (RUN lines 3 and 4) and UNSUPPORTED: cl_options
 - the second with full driver support (RUN lines 5-1) which should be supported by clang-cl.
The source code itself can be shared.

Reply:
Hi Vladimir, do you mean to have two separate tests with the same code but different RUN lines? that can be done but seems a bit redundant for this case as I came to realize that RUN lines 3-4 might not be necessary for this test. Initially I'd added this test inside another test (use_pinned_host_memory.cpp which had RUN lines 3-4) and added rest of the RUN lines specifically for this test for a specific case. please review the next patch and let me know if that's okay.